iconoir.com/components/Button.tsx

```diff
@@ -64,3 +68,21 @@ export const Button = styled(LargeButton)`
     background: var(--darker-gray);
   }
 `;
+
+export const CopyButton = styled(Button)`
+  text-transform: uppercase;
+  background: var(--white);
+  height: 30px;
+  padding: 0 12px;
+  font-size: 11px;
+  letter-spacing: 0.12em;
+  line-height: 17.6px;
+  font-weight: 700;
+  color: var(--black);
+  transition: background 0.1s linear, color 0.1s linear;
+  font-family: var(--font-family) !important;
+  &:hover {
+    background: var(--g0) !important;
+    color: var(--white);
+  }
```

Comment:
Do we really want this color transformation? The other buttons don't have it.

### Before

https://user-images.githubusercontent.com/5363448/226179145-e985a76a-972e-4284-9a05-a87a07e657e6.mp4

### After

https://user-images.githubusercontent.com/5363448/226179161-466b31e9-abb5-4ac2-862c-b79c70c5fc75.mp4

```suggestion
  font-family: var(--font-family) !important;
```

Reply:
Yeah, I agree. I wasn't against the effect while working on it, but it absolutely makes sense to merge it with all the other buttons 🙌